### PR TITLE
test: Remove unnecessary shadow root helpers

### PR DESCRIPTION
**Pull request by Artur-**

### CODE REVIEW

File: flow-tests/test-live-reload/src/test/java/com/vaadin/flow/uitest/ui/JavaLiveReloadIT.java [hunk 1]
```diff
@@ -69,17 +69,18 @@ public void splashMessageShownOnAutoReloadAndClosedOnBodyClick() {
 
         WebElement liveReload = findElement(By.tagName("vaadin-devmode-gizmo"));
```
Comment (caalador) [inline flow-tests/test-live-reload/src/test/java/com/vaadin/flow/uitest/ui/JavaLiveReloadIT.java:70]: Should be `TestBenchElement liveReload = $("vaadin-devmode-gizmo").first();`

File: flow-tests/test-live-reload/src/test/java/com/vaadin/flow/uitest/ui/JavaLiveReloadIT.java [hunk 2]
```diff
@@ -101,8 +102,8 @@ public void deactivateLiveReload() {
         // then: page is not reloaded
         WebElement liveReload2 = findElement(
```
Comment (caalador) [inline flow-tests/test-live-reload/src/test/java/com/vaadin/flow/uitest/ui/JavaLiveReloadIT.java:103]: `TestBenchElement liveReload2 = $("vaadin-devmode-gizmo").first();`

File: flow-tests/test-live-reload/src/test/java/com/vaadin/flow/uitest/ui/FrontendLiveReloadIT.java [hunk 3]
```diff
@@ -59,8 +59,7 @@ public void liveReloadOnTouchedFrontendFile() {
         // then: the frontend changes are visible in the DOM
         WebElement customComponent = findElement(
```
Comment (caalador) [inline flow-tests/test-live-reload/src/test/java/com/vaadin/flow/uitest/ui/FrontendLiveReloadIT.java:60]: `TestBenchElement customComponent = $(TestBenchElement.class).id(FrontendLiveReloadView.CUSTOM_COMPONENT);`

File: flow-tests/test-live-reload/src/test/java/com/vaadin/flow/uitest/ui/PreserveOnRefreshLiveReloadIT.java [hunk 4]
```diff
@@ -30,8 +30,8 @@ public void notificationShownWhenLoadingPreserveOnRefreshView() {
 
         WebElement liveReload = findElement(By.tagName("vaadin-devmode-gizmo"));
```
Comment (caalador) [inline flow-tests/test-live-reload/src/test/java/com/vaadin/flow/uitest/ui/PreserveOnRefreshLiveReloadIT.java:31]: `TestBenchElement liveReload = $("vaadin-devmode-gizmo").first();`

**Review by caalador — CHANGES_REQUESTED** [review]

File: flow-tests/test-live-reload/src/test/java/com/vaadin/flow/uitest/ui/JavaLiveReloadIT.java (hunk 2 above)
Comment (Artur-) [reply flow-tests/test-live-reload/src/test/java/com/vaadin/flow/uitest/ui/JavaLiveReloadIT.java]: Fixed

File: flow-tests/test-live-reload/src/test/java/com/vaadin/flow/uitest/ui/FrontendLiveReloadIT.java (hunk 3 above)
Comment (Artur-) [reply flow-tests/test-live-reload/src/test/java/com/vaadin/flow/uitest/ui/FrontendLiveReloadIT.java]: Fixed

File: flow-tests/test-live-reload/src/test/java/com/vaadin/flow/uitest/ui/PreserveOnRefreshLiveReloadIT.java (hunk 4 above)
Comment (Artur-) [reply flow-tests/test-live-reload/src/test/java/com/vaadin/flow/uitest/ui/PreserveOnRefreshLiveReloadIT.java]: Fixed

File: flow-tests/test-live-reload/src/test/java/com/vaadin/flow/uitest/ui/JavaLiveReloadIT.java (hunk 1 above)
Comment (Artur-) [reply flow-tests/test-live-reload/src/test/java/com/vaadin/flow/uitest/ui/JavaLiveReloadIT.java]: Fixed

**Review by caalador — APPROVED** [review]

**Pull request merged**